recurring_payments_stripe/models/sale_subscription.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import logging
+import stripe
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class SaleSubscription(models.Model):
+    _inherit = "sale.subscription"
+
+    is_recurrent = fields.Boolean(string="Pago Automático Recurrente")
```

Comment:
Im not sure if this variable name make really sense because a subscription is already recurrent (at least the invoice generation).  It needs to be changes everywhere.
```suggestion
    charge_automatically = fields.Boolean(string="Pago Automático Recurrente")
```